openerp-atomfeed-service/src/main/java/org/bahmni/feed/openerp/OpenERPAtomFeedProperties.java

```diff
@@ -71,9 +73,11 @@ public String getOdooURIForJob(Jobs feedJob, boolean isRestEnabled) {
         if (isRestEnabled) {
```

Comment:
I would have created two different methods instead called: getOdooXMLUriEndpoint() and getOdooRESTUriEndPoint(forFeed). The caller already knows whether its XML or Rest.. why keep passing the flag and put this logic here? Any advantage?

Reply:
Makes sense, will do.